app/components/marketing/hero-section.tsx

```diff
@@ -58,15 +58,16 @@ export default function HeroSection() {
             </div>
           </Form>
         </div>
-        <div className="w-4/5 mb-3 max-w-[600px] flex self-start">
+        <div className="w-4/5 mb-3 max-w-[600px] flex self-start m-4">
           <Image
             className="w-full"
-            src="/imgs/Hero-Image.png"
-            alt="group of people"
+            src={staticImage.largeGroup.url}
+            alt={staticImage.largeGroup.altText}
+            title={staticImage.largeGroup.title}
             height={379}
             width={553}
             priority
-          />
+          ></Image>
```

Comment:
Is this change intended/needed?

```suggestion
          />
```